python-lib/tagger.py

```diff
@@ -1,9 +1,110 @@
+"""Module to prepare the tagging of the documents
+splits on sentences and instanciates a Matcher to tag the documents"""
+
+from spacy_tokenizer import MultilingualTokenizer
+from dkulib_io_utils import generate_unique
+from spacy.matcher import PhraseMatcher
+from fastcore.utils import store_attr
+from typing import AnyStr
 import pandas as pd
 
 
 class Tagger:
-    def __init__(self, settings):
-        pass
+    def __init__(
+        self,
+        text_df,
+        ontology_df,
+        text_column,
+        language,
+        language_column,
+        tag_column,
+        category_column,
+        keyword_column,
+        lemmatize,
+        case_sensitive,
```

Comment:
Can we switch this parameter to `case_insensitive`? From a UX perspective, it would be more natural for users to have to "click" the option to simplify. Today we have "Case-sensitive" which simplifies if not clicked, versus Lemmatization and Unicode normalization which simplifies if clicked. Do you see what I mean?

Reply:
makes sense